itl/command-plugins.conf

```diff
@@ -465,6 +465,10 @@ object CheckCommand "http" {
 			value = "$http_post$"
 			description = "URL encoded http POST data"
 		}
+		"-P" = {
+			value = "$http_post_pw$"
+			description = "URL encoded http POST data, but Icingaweb2 should hide this one, as it contains a password."
+		}
```

Comment:
Hello @TQQEU!

I'm afraid this would break existing setups as the new -P would override the existing -P.

While you could add something like `"-P (password)" = ...` and override the key, I'd suggest to do something like this instead: https://github.com/Icinga/icinga2/pull/9758/commits/d3d74c237c54eed61d67600c2e04c3285a630189
Preferably just in your own setup, probably in your default service template.

Alternatively make http_post also a protected variable.

Best,
A/K

Reply:
Hello @Al2Klimov,

thanks for the suggestion of doing it like [d3d74c2].
I redid my code like this.

As long as this request is not merged, I already have an additional CheckCommand in my setup - basically I wanted to return a problem with it's respective solution with the side effect of ensuring that the configurations do not drift apart in the future.

If this is not useful for the community please decline my request / ask me to close my request.